app/lib/service/services.dart

```diff
@@ -177,7 +177,17 @@ Future<R> withFakeServices<R>({
     }
 
     // register fake services that would have external dependencies
-    registerAuthProvider(FakeAuthProvider());
+
+    // Local override for OAuth services when developing for external servers.
+    if (envConfig.fakeOauthSiteAudience != null &&
+        envConfig.fakeOauthSiteAudienceSecret != null) {
+      registerAuthProvider(DefaultAuthProvider(
+        oauthSiteAudience: envConfig.fakeOauthSiteAudience,
+        oauthSiteAudienceSecret: envConfig.fakeOauthSiteAudienceSecret,
```

Comment:
Why can't we put these in the config file?

Are we simply registering an oauth flow for use on localhost, or what is the plan?
Because I'm not sure it'd be horrible to registered that in the staging site, and check it into git in our config files.

Reply:
Removed these overrides.